deployment/ansible/roles/model-my-watershed.rwd/templates/upstart-mmw-rwd.conf.j2

```diff
@@ -0,0 +1,28 @@
+description "Rapid Watershed Delineation"
+
+start on (filesystem and started docker)
+stop on stopping docker
+
+kill timeout 20
+kill signal CONT
+respawn
+
+pre-start script
+  /usr/bin/docker kill docker-rwd || true
+  /usr/bin/docker rm docker-rwd || true
+end script
+
+exec /usr/bin/docker run \
+  --name docker-rwd \
```

Comment:
Nitpick, but I'd probably just call with `rwd` because the Docker part is kinda implied.